test/PushNotificationsTests/APITests.cpp

```diff
@@ -91,13 +91,65 @@ namespace Test::PushNotifications
             return true;
         }
 
-        TEST_METHOD(LaunchPackagedApp)
+        TEST_METHOD(ChannelRequestUsingNullRemoteId)
         {
             wil::unique_event event = CreateTestEvent(c_testProtocolScheme_Packaged);
 
             // This is associated protocol for the MSIX installed app for launch.
             // Use the ://path to define the component you want to test.
-            Uri launchUri{ c_testProtocolScheme_Packaged + L"://this_is_a_test" };
+            Uri launchUri{ c_testProtocolScheme_Packaged + L"://ChannelRequestUsingNullRemoteId" };
+            auto launchResult = Launcher::LaunchUriAsync(launchUri).get();
+            VERIFY_IS_TRUE(launchResult);
+
+            WaitForEvent(event, m_failed);
+        }
+
+        TEST_METHOD(ChannelRequestUsingRemoteId)
+        {
+            wil::unique_event event = CreateTestEvent(c_testProtocolScheme_Packaged);
+
+            // This is associated protocol for the MSIX installed app for launch.
+            // Use the ://path to define the component you want to test.
+            Uri launchUri{ c_testProtocolScheme_Packaged + L"://ChannelRequestUsingRemoteId" };
+            auto launchResult = Launcher::LaunchUriAsync(launchUri).get();
+            VERIFY_IS_TRUE(launchResult);
+
+            WaitForEvent(event, m_failed);
```

Comment:
Could all of this be parameterized in one function? All test functions follow the same pattern. The only thing that differs is the test URI string.